{
	"id": "openSUSE-SU-2025:14760-1",
	"modified": "2025-02-09T00:00:00Z",
	"published": "2025-02-09T00:00:00Z",
	"related": [
		"CVE-2025-0938"
	],
	"summary": "python313-3.13.2-1.1 on GA media",
	"details": "These are all security issues fixed in the python313-3.13.2-1.1 package on the GA media of openSUSE Tumbleweed.",
	"affected": [
		{
			"package": {
				"ecosystem": "openSUSE:Tumbleweed",
				"name": "python313",
				"purl": "pkg:rpm/opensuse/python313&distro=openSUSE%20Tumbleweed"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "3.13.2-1.1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"binaries": [
					{
						"python313": "3.13.2-1.1",
						"python313-32bit": "3.13.2-1.1",
						"python313-curses": "3.13.2-1.1",
						"python313-dbm": "3.13.2-1.1",
						"python313-idle": "3.13.2-1.1",
						"python313-tk": "3.13.2-1.1",
						"python313-x86-64-v3": "3.13.2-1.1"
					}
				]
			}
		}
	],
	"references": [
		{
			"type": "ADVISORY",
			"url": "https://lists.opensuse.org/archives/list/security-announce@lists.opensuse.org/thread/7NX5AR5RGDPJDJFRYBUSRNLERPCINRWF/"
		},
		{
			"type": "WEB",
			"url": "https://www.suse.com/security/cve/CVE-2025-0938"
		}
	]
}
